input/pagecontent/reference.md

```diff
@@ -185,13 +191,13 @@ FSH strings support the escape sequences that FHIR already defines as valid in i
+FHIR elements can contain [references to other Resources](https://hl7.org/fhir/R5/references.html#2.1.3.0). FSH represents references using the syntax `Reference({Resource/Profile})`. A resource or profile SHALL be identifiable by name, id, or URL. For example, `Reference(USCorePatientProfile)`, `Reference(us-core-patient)`, and `Reference(http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient)` all are valid references to the [US Core Patient profile](https://hl7.org/fhir/us/core/structuredefinition-us-core-patient.html). When referring to a Reference element, the `Reference()` MUST be included, except in the case of a [reference choice path](#reference-paths). When syntax allows for multiple References, the items are separated by `or` placed *inside* the parentheses, e.g. `Reference(Patient or Practitioner)`, **not** `Reference(Patient) or Reference(Practitioner)`. 
 
-In constructing profiles, references typically refer to resource or profile *types*, for example, the subject of an Observation could be constrained to `Reference(Patient or Group)`. Inside instances, references typically refer to other instances, for example, a subject of an Observation could be `Reference(JaneDoe)`, assuming JaneDoe names a Patient instance. In this case, since `JaneDoe` is a Patient instance, `Reference(JaneDoe)` is resolved to `Patient/JaneDoe`. If a reference value in an instance does not reference an existing instance, the value is used directly. For example, if the subject of an Observation is `Reference(Alice)`, and Alice does not name a Patient instance, the reference resolves to `Alice`.
+In constructing profiles, references typically refer to resource or profile *types*, for example, the subject of an Observation could be constrained to `Reference(Patient or Group)`. Inside instances, references typically refer to other instances, for example, a subject of an Observation could be `Reference(JaneDoe)`, assuming JaneDoe names a Patient instance. In this case, since `JaneDoe` is a Patient instance with id `jane-doe`, `Reference(JaneDoe)` is resolved to `Patient/jane-doe`. For example, if the subject of an Observation is `Reference(Alice)`, and Alice is not the name, id, or URL of a known Patient instance, the reference resolves to the literal string "Alice".
```

Comment:
Please review the resolutions for [FHIR-42616](https://jira.hl7.org/browse/FHIR-42616) and [FHIR-42306](https://jira.hl7.org/browse/FHIR-42306), as it seems they are not properly reflected here.

In the case of FHIR-42616, it looks like you implemented the balloter's original suggestion instead of the agreed upon resolution. In the case of FHIR-42306, it looks like you mistakenly deleted the sentence: "If a reference value in an instance does not reference an existing instance, the value is used directly."